doc/nrf/known_issues.rst

```diff
@@ -2411,20 +2443,21 @@ The issues in this section are related to components provided in :ref:`nrfxlib`.
 
 Crypto
 ======
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+The issues in this section are related to :ref:`nrfxlib:crypto`.
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 NCSDK-20688: For ChaCha-Poly, incorrect tag will be produced if plaintext is empty
```

Comment:
Wrong tracking issue. Please update to NCSDK-21781

Reply:
NCSDK-21781 is related to nRF5 SDK not nRF Connect SDK.

Reply:
I believe the correct jira is NCSDK-21430, just conforming with the team

Reply:
Thanks: NCSDK-21430 is correct.
This should be marked as fixed for 2.4.0